modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -183,85 +202,90 @@
 /datum/job/cyborg
 	alt_titles = list(
 		"Cyborg",
-		"Robot",
 		"Android",
+		"Machine",
+		"Robot",
 	)
 
 /datum/job/detective
 	alt_titles = list(
 		"Detective",
+		"Forensic Scientist",
 		"Forensic Technician",
 		"Private Investigator",
-		"Forensic Scientist",
 	)
 
 /datum/job/doctor
 	alt_titles = list(
 		"Medical Doctor",
-		"Surgeon",
-		"Nurse",
 		"General Practitioner",
 		"Medical Resident",
+		"Nurse",
 		"Physician",
+		"Surgeon",
+		"Medical Student",
```

Comment:
We already have Medical resident which is a junior intern-like role for MDs. Why add another alt title that also signifies apprentice doctors?

Reply:
A Resident is expected to know their job pretty well. Residency, to me, is just for RP flavor. The newbie titles are meant to CLEARLY outline who is actually, physically, new to the job so they can be focused on for teaching. Hence med student.